onnxruntime/core/language_interop_ops/torch/refcount_tracker.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <Python.h>
+#include <iostream>
+#include "core/language_interop_ops/torch/refcount_tracker.h"
+#include "core/platform/env.h"
+
+namespace onnxruntime {
+namespace language_interop_ops {
+namespace torch {
+#ifndef NDEBUG
+
+RefCountTracker::RefCountTracker() {
+  addr_info_map_ = {
+      {RefCountTracker::ObjCategory::ForwardArgs, forward_arg_addresses_},
+      {RefCountTracker::ObjCategory::ReturnValues, return_value_addresses_},
+      {RefCountTracker::ObjCategory::AutoGradContext, auto_grad_addresses_},
+  };
+}
+
+void RefCountTracker::TrackPyObject(RefCountTracker::ObjCategory category, PyObject* py_obj, std::string log_tag) {
+  AddressInfos& addrs = addr_info_map_[category];
+  void* addr = static_cast<void*>(py_obj);
+  auto it = addrs.find(addr);
+  if (it == addrs.end()) {
+    addrs.insert({addr, {log_tag}});
+  } else {
+    addrs[addr].push_back(log_tag);
+  }
+  std::cout << "Track" << ObjCategoryToString(category) << "\tAddress: [" << addr << "]\tRefCnt: " << Py_REFCNT(addr) << "\tLogTag: " << log_tag << std::endl;
```

Comment:
I don't remember seeing these messages while running... is this code used?  May be that I have just been running the forward pass.  In any case, we should shift to using the existing logging macros.

Reply:
Will use default logger to print the info. 

This is only for debug build when we want to compare the refcnt before and after some operations.